projects/onednn/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/oneapi-src/oneDNN"
+main_repo: "https://github.com/oneapi-src/oneDNN"
+language: c++
+vendor_ccs:
+- david@adalogics.com
+fuzzing_engines:
+- libfuzzer
```

Comment:
afl doesn't work?

Reply:
afl timed out in the CI -- see e.g. https://github.com/google/oss-fuzz/actions/runs/4925202744/jobs/8799144530 

Am not quite sure why the build time is so much more though.